core/src/Revolution/Processors/Browser/Browser.php

```diff
@@ -143,7 +143,7 @@ public function sanitize($file)
     {
         $file = rawurldecode($file);
         $file = strip_tags($file);
-        $file = preg_replace('#^[.\/]+#u', '', $file);
+        $file = preg_replace('#^(\.{2}|/)+#u', '', $file);
```

Comment:
```suggestion
        $file = preg_replace('#\.(?![\w\-\~])#u', '', $file);
        $file = preg_replace('#\/{2,}#', '/', $file);
```
This strategy here is to take care of the dots first, then simply reduce multiple slashes to single ones. The one thing to consider here is whether the class of allowable chars directly after a valid dot is flexible enough; I think so, but others should chime in if not.